TestDs3/Helpers/ProgressTrackers/TestJobItemTracker.cs

```diff
@@ -13,27 +13,35 @@
  * ****************************************************************************
  */
 
-using Ds3.Helpers.ProgressTrackers;
-using Ds3.Models;
-using NUnit.Framework;
 using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using Ds3.Helpers.ProgressTrackers;
+using Ds3.Models;
+using NUnit.Framework;
 
 namespace TestDs3.Helpers.ProgressTrackers
 {
     [TestFixture]
     public class TestJobItemTracker
     {
-        private static readonly IEnumerable<IEnumerable<Range>> _successes = new[]
+        private static readonly IEnumerable<IEnumerable<Range>> Successes = new[]
```

Comment:
I don't know that there is a single accepted convention.  You'll see both in the larger .net community.  My preference is to not use `_`